source/nodejs/README.md

```diff
@@ -29,10 +30,10 @@ We use lerna to manage package linking and building.
 # Start the documentation site (http://adaptivecards.io)
 
 1. `cd adaptivecards-site`
-2. `npm install hexo-cli -g`
-3. `hexo generate`
-4. `hexo server`
-5. `Open up the browser to point to localhost:[portnumber] printed after the command above returns under "Hexo is running at".`
+2. `npx lerna run release`
+3. `npm install hexo-cli -g`
+5. `hexo server`
+6. Open up the browser to point to `localhost:[portnumber]` printed after the command above returns under "Hexo is running at".
```

Comment:
Typo? Missing the backtick?

Reply:
Not sure what you mean. This is markdown syntax, and I didn't write this.

Reply:
Yes but it's changed and am trying to explain the changes - Looks like these files i am commenting on that you dont recognize the changes to should be reverted 

---
In reply to: [297809373](https://github.com/microsoft/AdaptiveCards/pull/2995#discussion_r297809373) [](ancestors = 297809373)

Reply:
I honestly don't know.